go.mod

```diff
@@ -63,6 +53,7 @@ require (
 )
 
 replace (
+	github.com/gardener/etcd-druid => github.com/georgekuruvillak/etcd-druid-1 v0.0.0-20200327150752-db8a2ba18426
```

Comment:
Shouldn't it be pointing commit on master instead of individual fork? You might have used it while development. But please keep gardener master  in sync with gardener/etcd-druid master rather than druid-fork/etcd-druid.